packages/pigment-css-nextjs-plugin/src/index.ts

```diff
@@ -55,10 +55,13 @@ export function withPigment(nextConfig: NextConfig, pigmentConfig?: PigmentOptio
           placeholderCssFile: extractionFile,
         },
         async asyncResolve(what: string, importer: string, stack: string[]) {
+          if (what.startsWith('__barrel_optimize__')) {
```

Comment:
Should this be combined with the `next/font` below?

Reply:
I did it knowingly since I don't know yet what kind of exports `__barrel_optimize__` expects. So I am just returning `next-font` for now.
In future, depending on any bug reports, it might require it's own dummy module similar to `next-font.js`.

Reply:
Got it, then I think it deserves a comment here.